src/chainstate/stacks/db/mod.rs

```diff
@@ -672,6 +673,7 @@ pub struct ChainStateBootData {
     pub first_burnchain_block_height: u32,
     pub first_burnchain_block_timestamp: u32,
     pub initial_balances: Vec<(PrincipalData, u64)>,
+    pub pox_constants: Option<PoxConstants>,
```

Comment:
I don't think this should be `Option<..>`.  I think the boot data should always have PoX constants.

Reply:
addressed this